README.md

```diff
@@ -5,16 +5,18 @@ modules and plugins that are common to all Ansible Network roles.  All of
 the artifacts in this role can be used independent of the platform that is
 being managed.
 
-Any open bugs and/or feature requests are tracked in [GitHub issues](../../issues).
+To use this role, install it from [Galaxy](https://galaxy.ansible.com/ansible-network/network-engine/). To find other roles maintained by the Ansible Network team, see our [Galaxy Profile](https://galaxy.ansible.com/ansible-network/).
 
-Interested in contributing to this role, please see [CONTRIBUTING](CONTRIBUTING.md)
+Any open bugs and/or feature requests are tracked in [GitHub issues](https://github.com/ansible-network/network-engine/issues).
+
+Interested in contributing to this role, please see [CONTRIBUTING](https://github.com/ansible-network/network-engine/blob/devel/CONTRIBUTING.md)
```

Comment:
I'd phrase this as a question - "Interested in contributing to this role? Check out [Contributing] before submitting a pull request."